### DISCUSSION TITLE
大佬们  ArrayTable怎么监听分页事件

### DISCUSSION THREAD

**Discussion by zhangguofei99** [alibaba/formily#3723] [Q&A 2.x]
![798ed3882d1fbc804d0208ef71a602a9](https://user-images.githubusercontent.com/96226678/219259364-f9930fa1-ee6d-40f5-bf37-7b309f63982b.png)
ArrayTable怎么监听分页变化 这样写没生效 应该怎么写呢 大佬们
https://codesandbox.io/s/boring-nova-pjodu6?file=/App.tsx

**Comment by janryWang** [comment]
什么场景需要监听分页？

**Reply by zhangguofei99** [reply]
操作记录
